src/devcenter/azext_devcenter/aaz/latest/devcenter/admin/network_connection/_list_health_check.py

```diff
@@ -12,20 +12,19 @@
 
 
 @register_command(
-    "devcenter admin network-connection list-health-detail",
-    is_preview=True,
+    "devcenter admin network-connection list-health-check",
```

Comment:
The command name changed. Please verify

Reply:
Confirmed that the API name should be changed